queries/ruby/scopeTypes.scm

```diff
@@ -0,0 +1,21 @@
+(comment) @comment
```

Comment:
This works as does the `@class` tag. I'm still working on the other ones, so still WIP.